qubole-aws-terraform-deployment/qubole-deployment/modules/network_infrastucture/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "deployment_suffix" {
+}
+
+variable "data_lake_project_region" {
+  default = "ap-southeast-1"
+}
+
+variable "public_ssh_key" {
+  default = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCeetBUHVCREiirJD/wIWbaQ6li04ywavNuwytCSIHw7O5XPKdx5AsuELQZ/fs86zPLrJahBVz+ACQdKAeV9aVSMLg/+O7BtNKxZDlL+Gaj+FlaOFiCoBSNmiLUFWoVFDa5rT9OD9d6rvxkAs7ldDWYTXbomqeCC1gESmEjKtjbDVxMHZHaLeSPXOHcqhcPE1rbBONN0ykzgtxmsSxeAhYGPSjPvarG8BFOzyI9RFqVm5tspdi3ShEs93uhFqfdFUizyAZwTGTfIB141Lan7wyvuB6MbVKGNF8aIwiSeZnPt63s/1imiCB8cYztdEnsEXPy428O0IsqAeLVW43ZY6Lh"
```

Comment:
Are these ssh keys specific to your account?

Reply:
noted. removed